zest/releaser/tests/vcs.txt

```diff
@@ -245,3 +245,18 @@ Setup.py with an uppercase VERSION:
     from setuptools import setup
     VERSION = '1.2'
     setup(name='urgh', version=VERSION)
+
+version in setup.cfg:
+
+    >>> setupcfgproject = os.path.join(tempdir, 'setupcfg')
+    >>> os.mkdir(setupcfgproject)
+    >>> os.chdir(setupcfgproject)
+    >>> lines = [
+    ...     "[metadata]",
+    ...     "  version = 1.0"]
```

Comment:
Remove the spaces in front of "version", this breaks the configparser on python 2.7 and on pypy. (That is where the test failure on this PR comes from).